docs/operation/training.rst

```diff
@@ -0,0 +1,235 @@
+=======================
+Training and Evaluation
+=======================
+
+
+
+Virtual Environment Setup
+=========================
+
+Effective containerization is important when it comes to running machine learning models as there can be conflicting dependencies. You can either use a Virtual Environment or Conda.
+
+Virtual Environment Installation and Setup
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+#. Install the virtual environment package:
+
+  .. code-block:: bash
+
+      $ sudo apt-get install python3-venv
+
+#. Create a virtual environment:
+
+  .. code-block:: bash
+
+      $ python3 -m venv ~/act  # Creates a venv "act" in the home directory, can be created anywhere
+
+#. Activate the virtual environment:
+
+  .. code-block:: bash
+
+      $ source act/bin/activate
+
+Conda Setup
+^^^^^^^^^^^
+
+#. Create a virtual environment:
+
+  .. code-block:: bash
+
+    $ conda create -n aloha python=3.8.10
+
+#. Activate the virtual environment:
+
+  .. code-block:: bash
+
+    $ conda activate aloha
+
+Install Dependencies
+^^^^^^^^^^^^^^^^^^^^
+
+Install the necessary dependencies inside your containerized environment:
+
+.. code-block:: bash
+
+  $  pip install dm_control==1.0.14
+  $  pip install einops
+  $  pip install h5py
+  $  pip install ipython
+  $  pip install matplotlib
+  $  pip install mujoco==2.3.7
+  $  pip install opencv-python
+  $  pip install packaging
+  $  pip install pexpect
+  $  pip install pyquaternion
+  $  pip install pyyaml
+  $  pip install rospkg
+  $  pip install torch
+  $  pip install torchvision
+
+Clone Repository
+================
+
+Clone ACT if using Aloha Stationary
+
+.. code-block:: bash
+
+    $ git clone https://github.com/Interbotix/act.git act_training_evaluation
```

Comment:
I would clarify that this command is cloning this to the directory `$(pwd)/act_training_evaluation`. If you want them to clone to their home directory, maybe do

```
$ cd ~
$ git clone https://github.com/Interbotix/act.git act_training_evaluation
```

or 

```
$ git clone https://github.com/Interbotix/act.git ~/act_training_evaluation
```